### PR TITLE
Add a README.md file

### PR DESCRIPTION
**Pull request by tannguyen16**
Add a README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Teradata Connector- Mule 4
+
+Anypoint Connector for Teradata (Teradata Connector) establishes communication between your Mule app and a Teradata relational database, enabling you to connect with your Teradata Vantage instance to load data and run SQL queries in Teradata tables.
```
Comment (adamtwo) [inline README.md]:
```suggestion
Anypoint Connector for Teradata (Teradata Connector) establishes communication between your Mule app and a Teradata Vantage database, enabling you to connect with your Teradata Vantage instance to load data and run SQL queries in Teradata Vantage tables.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+# Before You Begin
+
+To use this connector, you must be familiar with:
+
+- Teradata Vantage API
```
Comment (adamtwo) [inline README.md]:
```suggestion
- Teradata Vantage SQL
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+- Anypoint Connectors
+
+- Mule runtime engine (Mule)
+
+- Elements and global elements in a Mule flow
+
+- Creating a Mule app using Anypoint Studio (Studio)
```
Comment (adamtwo) [inline README.md]: The other elements on the list are nouns so I would change this one to a noun as well,e.g. :

```suggestion
- Anypoint Studio (Studio)
```

**Review by adamtwo — CHANGES_REQUESTED** [review]

**Comment by tannguyen16** [comment]
Hey Adam, I have updated the README file according to the comments! Please check once you have a chance. Thank you!

**Review by adamtwo — APPROVED** [review]

**Pull request merged**